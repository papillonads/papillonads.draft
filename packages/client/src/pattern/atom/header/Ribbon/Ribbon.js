import React from 'react'

const Ribbon = () => <div />

export default Ribbon
