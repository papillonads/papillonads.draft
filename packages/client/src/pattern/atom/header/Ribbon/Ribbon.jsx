import React from 'react'
import { string } from 'prop-types'
import cn from 'classnames'
import cs from './Ribbon.scss'

const Ribbon = ({ className }) => {
  const ribbonClassList = cn(className, cs.ribbon)

  return <div className={ribbonClassList} />
}

Ribbon.defaultProps = {
  className: '',
}

Ribbon.propTypes = {
  className: string,
}.isRequired

export default Ribbon
