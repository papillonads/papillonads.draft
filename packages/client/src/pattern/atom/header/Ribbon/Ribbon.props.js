import { string } from 'prop-types'

export const ribbonDefaultProps = {
  className: '',
}

export const ribbonPropTypes = {
  className: string,
}
