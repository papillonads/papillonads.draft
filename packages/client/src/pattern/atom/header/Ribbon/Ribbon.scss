@import '@primer/css/core/index.scss';

.ribbon {
  background-color: $orange-300;
  height: 5px;
}
