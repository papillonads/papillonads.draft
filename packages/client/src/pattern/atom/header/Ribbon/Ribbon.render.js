import React from 'react'
import Ribbon from './Ribbon'

const defaultRender = <Ribbon />

export { defaultRender }
