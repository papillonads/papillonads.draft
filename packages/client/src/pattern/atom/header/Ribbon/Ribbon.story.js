import { storiesOf } from '@storybook/react'
import { defaultRender } from './Ribbon.render'
import { getInfo } from '../../../../../../../.storybook/utility'

const defaultRenderInfo = getInfo('default')

storiesOf('Patttern/Atom/Header/Ribbon', module).add(defaultRenderInfo.text, () => defaultRender, defaultRenderInfo.parameters)
