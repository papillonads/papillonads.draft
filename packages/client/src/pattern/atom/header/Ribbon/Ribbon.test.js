import { defaultRender } from './Ribbon.render'

describe('<Ribbon />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
