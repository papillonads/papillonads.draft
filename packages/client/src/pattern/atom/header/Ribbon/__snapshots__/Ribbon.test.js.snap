// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Ribbon /> Snaphot must match defaultRender 1`] = `
<div
  className="ribbon"
/>
`;
